### ISSUE TITLE
Pages in storybook are no longer loading

### ISSUE THREAD

**Issue by alisonjoseph** [carbon-design-system/carbon-platform#1080]
![Screen Shot 2022-07-28 at 2 28 32 PM](https://user-images.githubusercontent.com/2753488/181621470-5def61ec-09d7-45f2-b722-0fd73041b58f.png)

**Comment by francinelucca** [comment]
Note: it'll work if you manually reload the browser after clicking on it 🤔

**Comment by mattrosno** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Development-624c3fc8b7972e001130fef7/issues/carbon-design-system/carbon-platform/1080?planning-poker) with ZenHub @alisonjoseph @andreancardona @francinelucca @jdharvey-ibm

**Comment by mattrosno** [comment]
@francinelucca @alisonjoseph @jdharvey-ibm will we have a separate Storybook for platform internal components like FrameworkIcon, CatalogItemMeta, etc.?

If not, this issue can probably be closed.

If we are having a separate Storybook for internal components, will we still include static pages? The thinking was that we could include static web pages in Storybook and then VRT against those, so we don't need a running web app for VRT.

If not, then we don't need full Next.js pages in Storybook (as cool as it is), and then this issue can be closed.

**Comment by alisonjoseph** [comment]
I don't think so, I think we will want to VRT against the actual web app, but will defer to @jdharvey-ibm to be sure.

**Comment by jdharvey-ibm** [comment]
Yeah, probably won't need internal components in a storybook for the purposes of VRT. Once we have static exports of the web-app, we can contrive any test pages we want and run VRT against those on `main` as well as on pull requests.

**Comment by jdharvey-ibm** [comment]
Maybe we use this issue to track the eventual removal of that storybook from the web-app?
